src/main/java/org/terasology/ligthandshadow/componentsystem/events/GameStartMessageEvent.java

```diff
@@ -0,0 +1,10 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+package org.terasology.ligthandshadow.componentsystem.events;
+
+import org.terasology.engine.entitySystem.event.Event;
+import org.terasology.engine.network.OwnerEvent;
+
+@OwnerEvent
+public class GameStartMessageEvent implements Event {
```

Comment:
if this is a notification event indicating that the pregame phase has started, why do we have `PregameEvent` that also indicates that the pregame phase has started :thinking:

Reply:
If I remember correctly this is run on the owner of the player whereas that is run on the player itself, we need to send the event to the owner so that the UI can be correctly configured and without this it doesn't work.

Reply:
ah okay, yeah that's possible of course.